docs/StardustDocs/topics/gettingStarted.md

```diff
@@ -2,157 +2,85 @@
+To use the Kotlin DataFrame library with the [custom Gradle configuration](gettingStartedGradle.md),
+[Jupyter Notebooks](gettingStartedJupyterNotebook.md) or [Datalore](gettingStartedJupyterNotebook.md),
+follow the instructions on the appropriate pages.
 
 ## Install Kotlin
 
 Kotlin is included in each IntelliJ IDEA release.
 Download and install [IntelliJ IDEA](https://www.jetbrains.com/idea/download/) to start using Kotlin.
 
-## Create Kotlin project
-
-1. In IntelliJ IDEA, select **File** | **New** | **Project**.
-2. In the panel on the left, select **New Project**.
-3. Name the new project and change its location, if necessary.
-
-   > Select the **Create Git repository** checkbox to place the new project under version control. You can enable this
-   > later at any time.
-   >
-   {type="tip"}
-
-4. From the **Language** list, select **Kotlin**.
-5. Select the **Gradle** build system.
-6. From the **JDK list**, select the [JDK](https://www.oracle.com/java/technologies/downloads/) that you want to use in
-   your project. The minimum supported version is JDK 8.
-    * If the JDK is installed on your computer, but not defined in the IDE, select **Add JDK** and specify the path to the
-      JDK home directory.
-    * If you don't have the necessary JDK on your computer, select **Download JDK**.
-7. From the **Gradle DSL** list, select **Kotlin** or **Groovy**.
-8. Select the **Add sample code** checkbox to create a file with a sample `"Hello World!"` application.
-9. Click **Create**.
-
-You have successfully created a project with Gradle.
-
-### Update Gradle dependencies
-
-In your Gradle build file (`build.gradle.kts`), add the Kotlin DataFrame library as a dependency:
 
+## Create your powerful application with a Kotlin DataFrame library
 <tabs>
-<tab title="Kotlin DSL">
-
-```kotlin
-dependencies {
-    implementation("org.jetbrains.kotlinx:dataframe:%dataFrameVersion%")
-}
-```
-
+<tab title="Gradle">
+Here is how you can take the first steps in developing Kotlin data-intensive applications with Gradle in IntelliJ IDEA.
+
+1. **Create your first application for data exploration:**
+   * To start from scratch,
+   create a [basic JVM application with the IntelliJ IDEA project wizard](gettingStartedGradle.md).
+   * To set up the custom Gradle configuration or Linter configuration, follow the instruction on the [page](gettingStartedGradleAdvanced.md).
+   * If you prefer more robust examples, try to use Kotlin DataFrame
+   together with KotlinDL like in the [Titanic example](https://github.com/Kotlin/dataframe/tree/master/examples/idea-examples/titanic).
+2. **Use Kotlin DataFrame and third-party Kotlin Data Science libraries in your application.**
+   * Find the curated list of the recommended science libraries on the JVM [here](https://kotlinlang.org/docs/data-science-overview.html#java-libraries).
+3. **Learn more about Kotlin DataFrame usage:**
```

Comment:
i was quite surprised that this page leads to a big blogpost with examples, video and bunch of links. Maybe it's worth it to pull out a video from there and embed it to the page, or provide short overview of the content?